packages/advanced-logic/src/extensions/payment-network/native-token.ts

```diff
@@ -38,4 +38,11 @@ export default abstract class NativeTokenPaymentNetwork extends ReferenceBasedPa
     }
     return super.createCreationAction(creationParameters);
   }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  protected isValidAddress(_address: string, _networkName?: string): boolean {
+    throw new Error(
+      `Default implementation of isValidAddress() does not support native tokens. Please override this method.`,
```

Comment:
Security to prevent running the default implementation of `isValidAddress()` on native chains which may have other address format.